src/timeresp.jl

```diff
@@ -117,8 +117,8 @@ function lsim(sys::AbstractStateSpace, u::AbstractVecOrMat, t::AbstractVector;
     if length(x0) != nx
         error("size(x0) must match the number of states of sys")
     end
-    if !(size(u) in [(length(t), nu) (length(t),)])
-        error("u must be of size (length(t), nu)")
+    if !(size(u) in [(nu, length(t)) (length(t),)]) # TODO check this and reshape automatically?
```

Comment:
```suggestion
    if !(size(u) in ((nu, length(t)), (length(t),))) # TODO check this and reshape automatically?
```
No need to allocate an array ;)